From 1dc8b5f6b306e24a66c5e999f7bface07fd2158f Mon Sep 17 00:00:00 2001
From: Daniel Walsh <dwalsh@redhat.com>
Date: Thu, 25 Feb 2010 14:24:00 +0300
Subject: [PATCH 02/11] SELinux relabeling

This patch adds a restorecon call to the init script to label the pid file
correctly.
---
 daemon-init.in |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/daemon-init.in b/daemon-init.in
index f2feb9d..75aa0b3 100644
--- a/daemon-init.in
+++ b/daemon-init.in
@@ -129,6 +129,7 @@ case "$1" in
 			rm -f $NagiosCommandFile
 			touch $NagiosRunFile
 			chown $NagiosUser:$NagiosGroup $NagiosRunFile
+			[ -x /sbin/restorecon ] && /sbin/restorecon $NagiosRunFile
 			$NagiosBin -d $NagiosCfgFile
                         pidof nagios > $NagiosRunFile
 			if [ -d $NagiosLockDir ]; then touch $NagiosLockDir/$NagiosLockFile; fi
-- 
1.7.3.5

